mandatory/2-function-creation.js

```diff
@@ -25,9 +37,18 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  // for (let i = 0; i < arrayOfStrings.length; i++) {
+  //   arrayOfStrings[i] = arrayOfStrings[i].trim().toLowerCase().replace("/", "");
+  // }
+  // return arrayOfStrings;
+  let newArray = arrayOfStrings.map(myFunction);
+  return newArray;
```

Comment:
Do you need a local variable here? It's a matter of style - since you immediately return the value, you may simplify the code and get rid of the variable:

```
return arrayOfStrings.map(myFunction);
```

Reply:
Thanks Dmitry, I did it.